code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -283,6 +286,12 @@
 				bleed_text += " incredibly quickly"
 
 			bleed_text += "!</B>\n"
+
+		for(var/i in bodyparts)
```

Comment:
```suggestion
		for(var/i in grasped_limbs)
```
This is what you intended here, right?
If so, checking `grasped_by` again is not necessary, right?